templates/postgresql.conf

```diff
@@ -0,0 +1,15 @@
+listen_addresses = '*'
```

Comment:
This is not what I meant. This config is still not overwriteable in any way. We need a variable for this.